docs/content/usage/trigger.en.md

```diff
@@ -15,6 +15,25 @@ $ fission ht create --url /hello --method GET --function hello
 trigger '94cd5163-30dd-4fb2-ab3c-794052f70841' created
 ```
 
+If you want to create a ingress for the HTTP trigger, you can provide the flag along with the hostname. If the hostname is not provided, it defaults to "*"
+
+```
+$ fission ht create --url /hello --method GET --function hello --createingress --host acme.com
+trigger '94cd5163-30dd-4fb2-ab3c-794052f70841' created
+
+$ fission route list
+NAME                                 METHOD HOST     URL      INGRESS FUNCTION_NAME
+94cd5163-30dd-4fb2-ab3c-794052f70841 GET    acme.com /hello   true    hello
+
+```
+
+Please note that for ingress to work, you will have to deploy a ingress controller in your cluster. Kubernetes currently supports and maintains for following ingress controller and you should check documentation of individual ingress controllers for setup.
```

Comment:
a ingress -> an ingress

Reply:
in your cluster -> to your Kubernetes cluster

Reply:
"Kubernetes currently supports and maintains for following ingress controller and you should check documentation of individual ingress controllers for setup." it is a bit of a weird sentence, maybe you can try rephrasing?

"Kubernetes currently supports and maintains the following ingress controllers:"